<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QWidget" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Formulario</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="info_lbl">
       <property name="text">
        <string>Selecione 2 itens que você almoçou e seus preços.</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="alm1_cbx">
         <item>
          <property name="text">
           <string>ovos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>misto quente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>queijo quente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>queijo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>homus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>iogurte</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>maçã</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>banana</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>laranja</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pão de quijo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cenouras</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pão</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>macarrão</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>tapioca</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>batatas fritas</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>café</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>refrigerante</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>água</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Café com Leite</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pre1R_sbx">
         <property name="specialValueText">
          <string/>
         </property>
         <property name="prefix">
          <string extracomment="R$">R$</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pre1C_sbx">
         <property name="prefix">
          <string extracomment=".">.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="alm2_cbx">
         <item>
          <property name="text">
           <string>ovos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>misto quente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>queijo quente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>queijo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>homus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>iogurte</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>maçã</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>banana</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>laranja</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pão de quijo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cenouras</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pão</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>macarrão</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>tapioca</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>batatas fritas</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>café</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>refrigerante</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>água</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pre2R_sbx">
         <property name="prefix">
          <string extracomment="R$">R$</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pre2C_sbx">
         <property name="prefix">
          <string>.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="total_lbl">
       <property name="text">
        <string>Total: R$</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
